src/bindings/python/tests/test_frontend/test_frontend_onnx_editor.py

```diff
@@ -346,6 +386,7 @@ def compare_functions(current, expected):  # noqa: C901 the function is too comp
         result = False
         msg += "Not equal number of ops. "
         msg += f"Current: {len(current_ops)}, expected: {len(expected_ops)}. "
+        return result
```

Comment:
If you just return early here, the error message won't be printed.

Reply:
Removed redundant return.